companydb_select_examples.sql

```diff
@@ -54,27 +70,42 @@ from works_on
 where hours between 25 and 30;
 
 
-
 -- 9. Display the names of projects that are based out of houston or stafford
 --      a) Use OR clause
 --      b) Use IN clause as in Plocation in ('Houston', 'Stafford')
 
 select * from dept_locations
 where dlocation in ('Houston','Stafford');
 
+select * from dept_locations 
+where dlocation='Houston' or dlocation='Stafford';
+
 -- 10. Display the names of the project that are neither based out of houston nor out of stafford
 --      a) Use AND/OR clause
 --      b) use NOT IN clause as in Plocation NOT IN ('Houston','Stafford')
 
+select *
+from project
+where plocation not in ('Houston','Stafford');
+
+select * 
+from project 
+where plocation <> 'Houston' and plocation <> 'Stafford';
+
```

Comment:
Not in would have been sufficient. Not need to use <> that why we are using not in.